apps/main/src/dex/components/PagePool/index.tsx

```diff
@@ -87,15 +87,15 @@ const Transfer = (pageTransferProps: PageTransferProps) => {
   const pricesApiPoolsMapper = useStore((state) => state.pools.pricesApiPoolsMapper)
   const fetchPricesPoolSnapshots = useStore((state) => state.pools.fetchPricesPoolSnapshots)
   const snapshotsMapper = useStore((state) => state.pools.snapshotsMapper)
-  const basePoolsLoading = useStore((state) => state.pools.basePoolsLoading)
   const { initCampaignRewards, initiated } = useStore((state) => state.campaigns)
 
   const storeMaxSlippage = useUserProfileStore((state) => state.maxSlippage[chainIdPoolId])
 
-  const { data: gaugeManager, isPending: isPendingGaugeManager } = useGaugeManager({
-    chainId: rChainId,
-    poolId: poolData?.pool.id!,
-  })
+  // Hidden until gauge manager is fixed
```

Comment:
I think it's better to delete the code. Or otherwise just create a `const isEnabled = false` and pass it as 2nd argument to the query to disable it